scene/gui/text_edit.cpp

```diff
@@ -6036,6 +6036,10 @@ void TextEdit::clear_executing_line() {
 	update();
 }
 
+void TextEdit::set_caret_on_readonly(bool p_value) {
```

Comment:
Maybe it should be `*_read_only` everywhere instead?

Reply:
You mean, snake case?
Hmm, I've just repeat how the readonly property is written so I don't know?
Particularly I like this way better.